### ISSUE TITLE
FLARE22 dataset split used for evaluation

### ISSUE THREAD

**Issue by Esoblu** [zongzi3zz/CAT#3]
Hi, thank you for sharing your great work!

I noticed that the datalist file in your repository doesn't include entries for the FLARE22 dataset. I would like to ask which split of the FLARE22 dataset was used for your evaluation — was it the official training set, the tuning set, or the test set?

**Comment by zongzi3zz** [comment]
We use the offine test set of 20 CT cases for evaluation.
